tools/sdk/uxdProtocol/createWithdrawInsuranceFromMangoDepositoryInstruction.ts

```diff
@@ -31,7 +30,7 @@ const createWithdrawInsuranceFromMangoDepositoryInstruction = async (
 
   return client.createWithdrawInsuranceFromMangoDepositoryInstruction(
     insuranceWithdrawnAmount,
-    { pda: getControllerPda(uxdProgramId) } as Controller,
+    instantiateController(uxdProgramId),
```

Comment:
Any other place in the code using a mocked Controller or Depository?

Reply:
I only used it in a few `create*Instruction` files of ours, it's all been replaced be `instantiateController`.